### PR TITLE
Add public notification components

### PR DESCRIPTION
**Pull request by jmhauck**
@MikeTschudi big one...was hoping we could get it started. We recently met with instant app folks for a design review so will have some more changes once we hear back. Not all doc is complete and the tests are not complete but has many of the necessary components.

### CODE REVIEW

**Review by MikeTschudi — CHANGES_REQUESTED** [review]
* Translations handling looks a lot better by being decentralized.
* Is there a to-do list for the components? Missing internal documentation, translations should be remedied once we move beyond the exploration phase.
* A number of the demos don't have license statements. The initial lines of a web page can be copied from, say, solution-contents.html.

**Comment by jmhauck** [comment]
@MikeTschudi no todo list aside from whats in my head. We can definitely get something going though.

File: src/components/buffer-tools/buffer-tools.tsx [hunk 1]
```diff
@@ -0,0 +1,210 @@
+import { Component, Element, Event, EventEmitter, Host, h, Prop, State, Watch } from '@stencil/core';
+import { loadModules } from "../../utils/loadModules";
+import Popover_T9n from '../../assets/t9n/buffer-tools/resources.json';
```
[the remark this answers is not in the document]
Comment (jmhauck) [reply src/components/buffer-tools/buffer-tools.tsx:3]: yes...an oversight...I'll update. I used the general naming convention they use in instant app components but this one should indicate buffer

File: src/components/config-map-search/config-map-search.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+              {
+                this.useLayerUrl ? 
+                  this._getUrlInput("Enter Layer Url", "layer-url") :
+                  this._getLayerListInput()
+              }
+            </calcite-tab>
+            <calcite-tab class="tab-padding">
+              <calcite-radio-button-group layout="vertical" name="locator-group">
+                <calcite-label layout="inline">
+                  <calcite-radio-button checked={this.useLocatorUrl} value="locator-url"/>From URL
```
[the remark this answers is not in the document]
Comment (jmhauck) [reply src/components/config-map-search/config-map-search.tsx:60]: I need to just delete those...the config components were something I started to look at some basic config options but that will basically come from the instant apps team. We may end up needing to do some focused config components but the general config we'll get from them.

File: src/assets/t9n/pdf-download/resources.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "pdfLabel": "{{n}} per page | {{s}}"
```
[the remark this answers is not in the document]
Comment (jmhauck) [reply src/assets/t9n/pdf-download/resources.json:2]: Sorry...I don't follow. Will reach out to chat

File: src/components/pdf-download/pdf-download.tsx [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+  _renderRemoveDuplicate(): VNode {
+    return this.removeDuplicateEnabled ? (
+      <div class="background-w padding-1-2 list-border margin-top-1">
+        <calcite-label layout="inline">
+          <calcite-switch
+            onCalciteSwitchChange={(evt) => this._removeDuplicatesChanged(evt)}
+            scale="m"
+          />
+          <span class="icon-text" title={"Remove duplicates by value"}>{"Remove duplicates by value"}</span>
```
[the remark this answers is not in the document]
Comment (jmhauck) [reply src/components/pdf-download/pdf-download.tsx:117]: We will filter out duplicate geometries but a desired function to filter by a field value has been brought up a couple of times. This is an oversight that it was still in here...I was just playing with the idea. Will clean it up.

File: src/assets/t9n/pdf-download/resources.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (jmhauck) [reply src/assets/t9n/pdf-download/resources.json:2]: updated {{s}} to {{labelSize}}

**Comment by jmhauck** [comment]
@MikeTschudi got at the above and added a bunch of license comments and updated them all to 2022

**Comment by MikeTschudi** [comment]
Ready to merge?

**Pull request merged**